plugins/module_utils/foreman_helper.py

```diff
@@ -364,6 +364,15 @@ def sanitize_entity_dict(entity_dict, name_map):
     return {value: entity_dict[key] for key, value in name_map.items() if key in entity_dict}
 
 
+# Helper for (global, operatingsystem, ...) parameters
```

Comment:
Can I be a real docstring please?

(Why is there no Pinocchio emoji?)

Reply:
Those comments in foreman_helper currently serve more as a structuring device for that file.